pepper-apis/dsm-core/src/main/java/org/broadinstitute/dsm/DSMServer.java

```diff
@@ -488,16 +488,37 @@ protected void setupDB(@NonNull Config config) {
 
         int maxConnections = config.getInt("portal.maxConnections");
         String dbUrl = config.getString("portal.dbUrl");
+        String dssDBUrl = config.getString("dss.dbUrl");
+        int dssMaxConnections = config.getInt("dss.maxConnections");//todo pegah add to SM
+        try {
+            TransactionWrapper.DbConfiguration DSS_DB =
+                    new TransactionWrapper.DbConfiguration(TransactionWrapper.DB.SHARED_DB, dssMaxConnections, dssDBUrl);
+            //setup the mysql transaction/connection utility
+            TransactionWrapper.init(DSS_DB);
+
+            logger.info("Running Shared DB update...");
+
+            LiquibaseUtil.runLiquibase(dssDBUrl, TransactionWrapper.DB.SHARED_DB);
+            LiquibaseUtil.releaseResources();
+
+            logger.info("Shared DB setup complete.");
+        } catch (Exception e) {
+            e.printStackTrace();
+        }
 
         //setup the mysql transaction/connection utility
-        TransactionWrapper.init(new TransactionWrapper.DbConfiguration(TransactionWrapper.DB.DSM, maxConnections, dbUrl));
+        TransactionWrapper.DbConfiguration DSM_DB =
+                new TransactionWrapper.DbConfiguration(TransactionWrapper.DB.DSM, maxConnections, dbUrl);
+        TransactionWrapper.init(DSM_DB);
 
-        logger.info("Running DB update...");
+        logger.info("Running DB update on DSM...");
 
         LiquibaseUtil.runLiquibase(dbUrl, TransactionWrapper.DB.DSM);
         LiquibaseUtil.releaseResources();
 
         logger.info("DB setup complete.");
+
+        TransactionWrapper.setDefaultDB(TransactionWrapper.DB.DSM);
```

Comment:
Setting a default database for DSM so all our old classes and queries that use DSM DB will work